### PR TITLE
German Credit Rewrite Basic

### PR DESCRIPTION
**Pull request by pkopper**

### CODE REVIEW

File: _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd [hunk 1]
```diff
@@ -452,43 +659,41 @@ names(pred_rf)
...
+| Package | Functionality |
+| :-      | :---------------------------------------------------------------------------- |
+| `mlr3`  | Framework for machine learning: `Task`, `Learner`, `resample()` and `benchmark()` |
+| `mlr3learners` | Concrete `Learner`s for many popular machine learning implementations |
+| `mlr3pipelines` | Dataflow programming of machine learning workflows. |
+| `mlr3tuning` | Hyperparameter tuning for machine learning algorithms. |
+| `mlr3filter` | Feature filtering |
+| `mlr3viz` | Visualisations and plots |
+| `paradox` | Auxiliary package providing (hyper)parameter handling |
+| `mlr3misc` | Auxiliary functions |
```
Comment (pat-s) [inline _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd]: Was this on purpose? Beforehand it was cleanly formatted, now its messy.

File: _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd [hunk 2]
```diff
@@ -452,43 +659,41 @@ names(pred_rf)
...
 
 ### Packages
 
 The non-`mlr3` packages we use:
 
-| Package        | Reason                                                                                                                                                                                                   |
```
Comment (pat-s) [inline _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd:478]: The last comment applies to all tables.

File: _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd [hunk 3]
```diff
@@ -80,34 +74,35 @@ theme_set(theme_light())
...
-| residence_since        | present residence since                                |
-| property_magnitude     | properties that applicant has                          |
-| age                    | age in years                                           |
-| other_payment_plans    | other installment plans the applicant is paying        |
-| housing                | type of apartment rented, owned, for free / no payment |
-| existing_credits       | number of existing credits at this bank                |
-| job                    | current job information                                |
-| num_dependents         | number of people being liable to provide maintenance   |
-| own_telephone          | is there any telephone registered for this customer?   |
-| foreign_worker         | is applicant foreign worker?                           |
```
Comment (pat-s) [inline _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd:110]: Does this really look cleaner now?

File: _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd [hunk 4]
```diff
@@ -142,7 +142,7 @@ skimr::skim(credit)
 library(mlr3misc)
 credit_short = map_dtc(credit, function(x) {
   if (is.factor(x) || is.character(x)) {
-    levels(x) = abbreviate(stringr::str_trunc(levels(x), 16), 12)
+    levels(x) = abbreviate(stringr::str_trunc(levels(x), 16) , 12)
```
Comment (pat-s) [inline _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd]: why a whitespace here?

File: _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd [hunk 5]
```diff
@@ -163,192 +163,301 @@ DataExplorer::plot_histogram(credit, nrow = 2, ncol = 3)
...
 ```{r}
 learner_rf = lrn("classif.ranger", importance = "permutation")
 learner_rf$train(task)
 ```
 
-- We can access the importance values using `$importance()`
+We can access the importance values using `$importance()`.
+The `R6` importance method plots the feature importance derived during training.
 
-```{r, R.options=list(width = 120)}
```
Comment (pat-s) [inline _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd:238]: I think this was there on purpose?

File: _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd [hunk 6]
```diff
@@ -163,192 +163,301 @@ DataExplorer::plot_histogram(credit, nrow = 2, ncol = 3)
...
 To measure the performance of a learner on new unseen data, we usually mimic the scenario of unseen data by splitting up the data into training and test set.
 The training set is used for training the learner, and the test set is only used for predicting and evaluating the performance of the trained learner.
-Many resampling methods (cross-validation, bootstrap) repeat the splitting process in different ways.
+Numerous resampling methods (cross-validation, bootstrap) repeat the splitting process in different ways.
 
-- We need to specify the resampling scheme using the `rsmp()` function:
+We need to specify the resampling strategy using the `rsmp()` function:
 
 ```{r}
-resampling = rsmp("holdout", ratio = 2 / 3)
```
Comment (pat-s) [inline _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd:317]: this whitespace is correct

File: _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd [hunk 7]
```diff
@@ -357,62 +466,164 @@ msr_list = list(
...
+This does **not work** on rstudio cloud, though.
+So only uncomment it locally!
 
-```{r, eval = FALSE}
+```{r}
 # future::plan("multiprocess")
 ```
 
-```{r}
+```{r}
```
Comment (pat-s) [inline _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd]: why a whitespace here?

**Comment by pat-s** [comment]
@pkopper Highly appreciated if this could be finished up soon :)

File: _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd (hunk 5 above)
Comment (pkopper) [reply _posts/2020-03-11-basics-german-credit/2020-03-11-mlr3-basics-german-credit.Rmd:238]: What do you mean?

**Comment by sumny** [comment]
see also mlr-org/mlr3gallery#59
